<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowser</class>
 <widget class="QWidget" name="FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10" rowstretch="1,1,20">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,0,0,0,0,0">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="3">
         <widget class="QToolButton" name="toParentButton">
          <property name="toolTip">
           <string>Parent Directory</string>
          </property>
          <property name="accessibleName">
           <string>Parent Directory</string>
          </property>
          <property name="accessibleDescription">
           <string>Go to the parent directory</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/arrow-up.png</normaloff>:/Resources/arrow-up.png</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+Up</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="forwardButton">
          <property name="toolTip">
           <string>Forward</string>
          </property>
          <property name="accessibleName">
           <string>Forward</string>
          </property>
          <property name="accessibleDescription">
           <string>Go forward</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/arrow-right.png</normaloff>:/Resources/arrow-right.png</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+Right</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="lookInCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QToolButton" name="btnRefresh">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/spinner.png</normaloff>:/Resources/spinner.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="backButton">
          <property name="toolTip">
           <string>Back</string>
          </property>
          <property name="accessibleName">
           <string>Back</string>
          </property>
          <property name="accessibleDescription">
           <string>Go back</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/arrow-left.png</normaloff>:/Resources/arrow-left.png</iconset>
          </property>
          <property name="shortcut">
           <string>Alt+Left</string>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="tbCut">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>剪切</string>
          </property>
          <property name="text">
           <string>剪切</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/cut.png</normaloff>:/Resources/cut.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="tbPaste">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>粘贴</string>
          </property>
          <property name="text">
           <string>粘贴</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/paste.png</normaloff>:/Resources/paste.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="tbCopy">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>复制</string>
          </property>
          <property name="text">
           <string>复制</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/copy.png</normaloff>:/Resources/copy.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="tbRename">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>重命名</string>
          </property>
          <property name="text">
           <string>重命名</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Resources/rename.png</normaloff>:/Resources/rename.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTreeView" name="tvNavigation">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageFiles">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeView" name="tvFiles">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="itemsExpandable">
              <bool>false</bool>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="headerHidden">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageFilteredFiles">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeView" name="tvFilteredFiles">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageProcessInfo">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="btnStop">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Resources/stop.png</normaloff>:/Resources/stop.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="plainTextEdit"/>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
